src/neural/network_demux.cc

```diff
@@ -235,8 +235,12 @@ void DemuxingComputation::ComputeBlocking() {
   dataready_cv_.wait(lock, [this]() { return dataready_ == 0; });
 }
 
-std::unique_ptr<Network> MakeDemuxingNetwork(const WeightsFile& weights,
-                                             const OptionsDict& options) {
+std::unique_ptr<Network> MakeDemuxingNetwork(
```

Comment:
Technically these ones could defer failure to the child networks, but I'm fine with it not - given the unlikelyhood of people using a roundrobin/demux/mux of random backends more than once...

Reply:
I wanted to have a first version ready quickly.

Reply:
update looks good.